src/wallet.rs

```diff
@@ -43,7 +43,7 @@ enum WalletSyncStatus {
 	InProgress { subscribers: tokio::sync::broadcast::Sender<Result<(), Error>> },
 }
 
-pub struct Wallet<D, B: Deref, E: Deref, L: Deref>
+pub(crate) struct Wallet<D, B: Deref, E: Deref, L: Deref>
```

Comment:
Visibility control for `Walle` and `WalletKeysManager` (below) seem unrelated to to the commits purpose.